tests/Api/Responses/Expected/shop/get_channel_by_code.json

```
@@ -0,0 +1,18 @@
+{
+    "@context": "\/api\/v2\/contexts\/Channel",
+    "@id": "\/api\/v2\/shop\/channels\/WEB",
+    "@type": "Channel",
+    "baseCurrency": {
+        "@id": "\/api\/v2\/shop\/currencies\/USD",
+        "@type": "Currency",
+        "code": "USD"
+    },
```

It looks wrong in comparison to the "defaultLocale". Why do we have IRI in one place and object in another? 
```suggestion
    "baseCurrency": "\/api\/v2\/shop\/currencies\/USD",
```

Also, I've requested IRI explicitly here